logging/deployment/src/main/java/io/quarkiverse/googlecloudservices/logging/deployment/LoggingBuildSteps.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkiverse.googlecloudservices.logging.deployment;
+
+import com.google.auth.oauth2.GoogleCredentials;
+
+import io.quarkiverse.googlecloudservices.logging.runtime.JsonFormatter;
+import io.quarkiverse.googlecloudservices.logging.runtime.LabelExtractor;
+import io.quarkiverse.googlecloudservices.logging.runtime.LoggingConfiguration;
+import io.quarkiverse.googlecloudservices.logging.runtime.TraceInfo;
+import io.quarkiverse.googlecloudservices.logging.runtime.TraceInfoExtractor;
+import io.quarkiverse.googlecloudservices.logging.runtime.ecs.EscJsonFormat;
+import io.quarkiverse.googlecloudservices.logging.runtime.format.JsonHandler;
+import io.quarkiverse.googlecloudservices.logging.runtime.format.TextHandler;
+import io.quarkiverse.googlecloudservices.logging.runtime.recorder.LoggingHandlerFactory;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.LevelTransformer;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.SimpleFormatter;
+import io.quarkiverse.googlecloudservices.logging.runtime.util.StackTraceArrayRenderer;
+import io.quarkus.arc.deployment.UnremovableBeanBuildItem;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.LogHandlerBuildItem;
+
+public class LoggingBuildSteps {
+
+    private static final String FEATURE = "google-cloud-logging";
+
+    @BuildStep
+    public FeatureBuildItem feature() {
+        return new FeatureBuildItem(FEATURE);
+    }
+
+    @BuildStep
+    public UnremovableBeanBuildItem credentials() {
+        return UnremovableBeanBuildItem.beanTypes(GoogleCredentials.class);
+    }
+
```

Comment:
I added it because it was removed at some point. But that might have been my local setup (same with the other unremovable beans). Will test again without it.